hasher-matcher-actioner/version.txt

```diff
@@ -1 +1 @@
-1.0.8
```

Comment:
let me know if it's preferred to split out the version bump

Reply:
I think it's okay for this case